Swift/Database.swift

```diff
@@ -591,4 +599,24 @@ extension CBLDatabase {
             }
         })
     }
+    
+    /// The same as `inBatch(() -> Void)`, but the closure can return a Bool.
+    /// If the closure returns `false`, the transaction will be aborted, and this function will return false.
+    internal func maybeBatch(block: () throws -> Bool) throws -> Bool {
+        var err: NSError? = nil
+        guard __maybeBatch(&err, usingBlockWithError: { (errPtr) -> Bool in
```

Comment:
Where is this __maybeBatch defined?

Reply:
This is because of the NS_REFINED_FOR_SWIFT on the `maybeBatch` function in CBLDatabase+Swift.h